retrieval_service/sql-proxy/cloudsql.tests.cloudbuild.yaml

```diff
@@ -0,0 +1,161 @@
+        "-r",
+        "requirements-test.txt",
+        "--user",
+      ]
+
+  - id: Create database
+    name: ${_IMAGE_NAME}-proxy
+    dir: retrieval_service/sql-proxy
+    entrypoint: /bin/bash
+    secretEnv:
+      - CLOUDSQLUSER # Use built-in env vars for database connection
+      - CLOUDSQLPASSWORD
+    args:
+      - "-c"
+      - |
+        /cloudsql/cloud-sql-proxy --port ${_DATABASE_PORT} ${_INSTANCE_CONNECTION_NAME} & sleep 2;
+        psql -h ${_DATABASE_HOST} -c "CREATE DATABASE ${_DATABASE_NAME};" || echo "Database '${_DATABASE_NAME}' already exists."
+        psql -h ${_DATABASE_HOST} -d ${_DATABASE_NAME} -c 'CREATE EXTENSION vector;' || echo "Extension 'vector' already exists."
+
+  - id: Initialize data
+    name: python:3.11
+    dir: retrieval_service
+    secretEnv:
+      - CLOUDSQLUSER
+      - CLOUDSQLPASSWORD
+    entrypoint: /bin/bash
+    args:
+      - "-c"
+      - |
+        # Create config
+        cp example-config.yml config.yml
+        sed -i "/127.0.0.1/d" config.yml
+        sed -i "s/postgres/cloudsql-postgres/g" config.yml
+        sed -i "s/my_database/${_DATABASE_NAME}/g" config.yml
+        sed -i "s/my-user/$$CLOUDSQLUSER/g" config.yml
+        sed -i "s/my-password/$$CLOUDSQLPASSWORD/g" config.yml
+        echo -e '\n  project: "'$PROJECT_ID'"' >> config.yml
+        echo '  region: "'${_CLOUDSQL_REGION}'"' >> config.yml
+        echo '  instance: "'${_CLOUDSQL_INSTANCE}'"' >> config.yml
+        # Run script
+        python run_database_init.py
+
+  - id: Run Alloy DB integration tests
+    name: python:3.11
+    dir: retrieval_service
+    env: # Set env var expected by tests
+      - "DB_NAME=${_DATABASE_NAME}"
+      - "DB_HOST=${_DATABASE_HOST}"
+      - "DB_PROJECT=$PROJECT_ID"
+      - "DB_REGION=${_CLOUDSQL_REGION}"
+      - "DB_INSTANCE=${_CLOUDSQL_INSTANCE}"
+    secretEnv:
+      - CLOUDSQLUSER
+      - CLOUDSQLPASSWORD
+    entrypoint: /bin/bash
+    args:
+      - "-c"
+      - |
+        # Set env var expected by tests
+        export DB_USER=$$CLOUDSQLUSER
+        export DB_PASS=$$CLOUDSQLPASSWORD
+        python -m pytest -m "not postgres"
+
+  - id: Run database export
+    name: python:3.11
+    dir: retrieval_service
+    entrypoint: /bin/bash
+    args:
+      - "-c"
+      - |
+        # Run script
+        python run_database_export.py
+        cd ../data
+        diff --strip-trailing-cr -Z airport_dataset.csv airport_dataset.csv.new || (echo "airport dataset export fail" && exit 1)
+        diff --strip-trailing-cr -Z amenity_dataset.csv amenity_dataset.csv.new || (echo "amenity dataset export fail" && exit 1)
+        diff --strip-trailing-cr -Z flights_dataset.csv flights_dataset.csv.new || (echo "flight dataset export fail" && exit 1)
+
+  - id: Clean exported files
+    name: python:3.11
+    dir: data
+    entrypoint: /bin/bash
+    args:
+      - "-c"
+      - |
+        rm airport_dataset.csv.new amenity_dataset.csv.new flights_dataset.csv.new
```

Comment:
nit: this is not needed. Since it's not stored in Google (like in Cloud Storage or another service), it will be cleaned up automatically after the test run.

Reply:
resolving this in another PR. (remove from the alloydb workflow)